pkg/sites/comicextra/comicextra.go

```diff
@@ -27,7 +27,41 @@ func retrieveImageLinks(comic *core.Comic) ([]string, error) {
 	}
 
 	return links, err
+}
+
+func RetrieveIssueLinks(url string) ([]string, error) {
+	if isSingleIssue(url) {
+		return []string{url}, nil
+	}
+
+	var links []string
+	name := util.SplitURL(url)[4]
+	set := make(map[string]struct{})
+
+	response, err := soup.Get(url)
+	if err != nil {
+		return nil, err
+	}
+
+	re := regexp.MustCompile("<a[^>]+href=\"([^\">]+" + "/" + name + "/.+)\"")
+	match := re.FindAllStringSubmatch(response, -1)
+
+	for i := range match {
+		url := match[i][1] + "/full"
+		if util.IsURLValid(url) {
+			set[url] = struct{}{}
```

Comment:
I think you can simply do
```go
links = append(links, url)
```
and avoid to use the `set`

Reply:
I think for other sites it is possible, but here since it is just using the regex there are duplicate links. Should I switch to using the soup doc here too?

Reply:
Using the regex is the best way here, with soup I think that you have to do a lot of extra checks.
About duplicates use `IsValueInSlice`:
```go
		if util.IsURLValid(url) && !util.IsValueInSlice(url, links) {
			links = append(links, url)
		}
```